recipe/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# from Continuum's build.sh
+
+LIB=$PREFIX/lib
+mkdir $LIB
+cd $LIB
+
+if [ $ARCH == '64' ]; then
+    cp /usr/local/lib64/libgfortran.so.3.0.0 .
```

Comment:
So Continuum's build.sh was based on a custom built GCC I made in a docker image a while back.  In general, we should never depend on system installed gfortran or libgfortran. It just makes teaching versions a lot harder. @mingwandroid and I hope to fix this with new GCC packages.  Any day now...